konfidency/utils/client.ts

```diff
@@ -8,4 +8,12 @@ export interface API {
       pageSize: number;
     };
   };
+
+  "POST /:customer/:sku/review": {
+    searchParams: {
+      sku: string;
+      customer: string;
+    };
```

Comment:
```suggestion
    response: ResponseWriteReview
```

The searchParams is for the query parameters that the endpoint handle.
Type the response here